src/client/pages/ToeristischeVerhuurDetail/VergunningVerhuur.tsx

```diff
@@ -68,19 +56,18 @@ function useStatusLineItems(
         documents: [],
         isActive:
           vergunning.caseType === 'B&B - vergunning'
-            ? vergunning.status !== 'Afgehandeld'
+            ? vergunning.status === 'Behandelen aanvraag'
```

Comment:
This can't be right. We should nit show a status like that in the Status/Process component. It should be "In behandeling". Will this status leak to the user?